.github/workflows/ci.yml

```diff
@@ -61,3 +61,33 @@ jobs:
         with:
           user: __token__
           password: ${{ secrets.PYPI_API_TOKEN }}
+
+  build_push:
+    name: Package docker image
+    runs-on: ubuntu-18.04
+    needs: ["checks", "release"]
+    if: "success('checks') && !failure('release')"
+    steps:
+      #- uses: actions/checkout@v2
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      # build release image ----
+      - name: "Release: Build and push"
+        uses: docker/build-push-action@v2
+        if: ${{ github.event_name == 'release' &&  startsWith(github.event.release.tag_name, 'hub') }}
+        with:
+          push: true
+          tags: ghcr.io/${{github.repository}}:${{ github.event.release.tag_name }}
+
+      # build any image pushed on a branch starting with development ----
+      - name: "Development: Build and push"
+        uses: docker/build-push-action@v2
+        if: ${{ github.event_name != 'release' && startsWith(github.ref_name, 'development') }}
```

Comment:
For posterity, the events that this workflow responds to are defined [here](https://github.com/cal-itp/calitp-py/blob/b088793c3d6cdc009707b55e71b3e9329293cf1e/.github/workflows/ci.yml#L4-L6), so for now specifying `event_name != 'release'` essentially means `event_name == 'push'`.